### PR TITLE
KS-SK1129-WHGR-Update-htaccess_protect_directory_by_password-for-Webhosting-TRANSLATION

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations are done

need proof for all subs except FRFR and FRCA, will need duplication for EN subs.

### CODE REVIEW

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md [hunk 1]
```diff
@@ -1,156 +1,269 @@
...
-updated: 2021-09-20
+updated: 2023-06-01
 ---
 
 > [!primary]
 > Esta traducción ha sido generada de forma automática por nuestro partner SYSTRAN. En algunos casos puede contener términos imprecisos, como en las etiquetas de los botones o los detalles técnicos. En caso de duda, le recomendamos que consulte la versión inglesa o francesa de la guía. Si quiere ayudarnos a mejorar esta traducción, por favor, utilice el botón «Contribuir» de esta página.
 >
 
-**Última actualización: 20/09/2021**
+**Última actualización: 01/06/2023"**
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md]:
```suggestion
**Última actualización: 01/06/2023**

```

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md [hunk 2]
```diff
@@ -1,156 +1,269 @@
...
+**Descubra cómo proteger un repertorio o el acceso a la parte de administrador de su sitio web mediante autenticación con los archivos ".htaccess" y ".htpasswd".**
 
 ## Requisitos
 
-- Tener un [plan de hosting](https://www.ovhcloud.com/es/web-hosting/).
-- Haber iniciado sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es).
-- Disponer de las claves necesarias para conectarse al [espacio de almacenamiento del alojamiento](../conexion-espacio-almacenamiento-ftp-alojamiento-web/).
+- Tener un [plan de hosting](https://www.ovhcloud.com/es-es/web-hosting/).
+- Estar conectado a su [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es).
+- Disponer de las claves de conexión a [el espacio FTP de su alojamiento](/pages/web/hosting/ftp_connection).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md]:
```suggestion
- Disponer de las claves de conexión al [espacio FTP de su alojamiento](/pages/web/hosting/ftp_connection).

```

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md [hunk 3]
```diff
@@ -1,156 +1,269 @@
...
 ## Procedimiento
 
 > [!primary]
 >
-> La solución que ofrece es solo una posibilidad técnica entre otras para crear un espacio de administrador en su sitio web. También puede utilizar la funcionalidad [Módulo en 1 clic](../modulos-en-un-clic/) que ofrece [OVHcloud](https://www.ovhcloud.com/es-es/).
+> La solución de seguridad propuesta aquí es solo una posibilidad técnica entre otras. 
+>
+> Tenga en cuenta, por ejemplo, que si utiliza un **C**onent **M**anagement **S**ystem (**CMS**), existen otras soluciones de seguridad.
+>
+Si utiliza un CMS WordPress, OVHcloud también pone a su disposición un tutorial sobre cómo [utilizar el archivo htaccess con WordPress](/pages/web/hosting/htaccess_how_to_protect_wordpress).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md]:
```suggestion
Si utiliza un CMS WordPress, OVHcloud también pone a su disposición un tutorial sobre [cómo utilizar el archivo htaccess con WordPress](/pages/web/hosting/htaccess_how_to_protect_wordpress).

```

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md [hunk 4]
```diff
@@ -1,156 +1,269 @@
...
+>
+> Para ello, contacte con nuestra [comunidad de usuarios](https://community.ovh.com/en/) o nuestros [partners de OVHcloud](https://partner.ovhcloud.com/es-es/directory/).
 >
 
-### Etapa 1: crear el árbol
+A continuación explicamos los 4 pasos principales que debe seguir para proteger el acceso a un directorio o a la totalidad o parte de su sitio web:
 
-Conéctese al [espacio de almacenamiento](../conexion-espacio-almacenamiento-ftp-alojamiento-web/) de su alojamiento. Abra el ["Pasta raiz"](../configurar-un-multisitio-en-un-alojamiento-web/#21-anadir-un-dominio-registrado-con-ovhcloud) de su sitio web.<br>
-Cree un archivo "crypter.php".
+- Crear archivos "crypt.php", ".htaccess" y ".htpasswd";
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md]:
```suggestion
- Crear archivos "crypt.php", ".htaccess" y ".htpasswd".

```

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md [hunk 5]
```diff
@@ -1,156 +1,269 @@
...
+
+> [!warning]
+>
+> Los pasos que se indican a continuación optimizarán la seguridad de los datos alojados.
+> Por lo tanto, si sus sitios web son compatibles, le recomendamos encarecidamente que utilice la versión de PHP más reciente posible.
+> 
+> Para modificar la versión de PHP de sus sitios web en su alojamiento web, consulte las siguientes guías:
+> 
+> - [Modificar la configuración de un alojamiento web](/pages/web/hosting/ovhconfig_modify_system_runtime)
+> - [Cambiar la versión de PHP de un alojamiento web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md]:
```suggestion
> - [Cambiar la versión de PHP de un alojamiento web](/pages/web/hosting/ovhconfig_modify_system_runtime)

```

Comment (OVH-Team-Guides) [reply pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md]: Please fix this for all versions

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md [hunk 6]
```diff
@@ -1,156 +1,269 @@
...
+Abra o cree la carpeta destinada a la protección de su sitio web. En nuestro ejemplo, esta es la carpeta "admin". Cree un archivo ".htpasswd" y un archivo ".htaccess" en este directorio.
 
 ![folder_admin](images/folder_admin.png){.thumbnail}
 
+Para utilizar correctamente los archivos ".htaccess" y ".htpasswd", debe conocer y respetar las siguientes reglas: 
+
+- **un único** fichero ".htaccess" y **un único** fichero ".htpasswd" por directorio o subdirectorio, para evitar conflictos entre diferentes ficheros ".htaccess" y diferentes ".htpasswd";
+- los archivos ".htaccess" y ".htpasswd" son invisibles para los internautas que visitan su sitio web;
+- las reglas declaradas en un archivo ".htaccess" se aplican a todo el directorio en el que está instalado el archivo ".htaccess", así como a todos los subdirectorios del mismo directorio.
+Los archivos ".htpasswd" y ".htaccess" pueden estar en carpetas diferentes. Solo puede utilizar un fichero ".htpasswd" para varios ".htaccess".
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_protect_directory_by_password/guide.es-es.md]:
```suggestion
- Los archivos ".htpasswd" y ".htaccess" pueden estar en carpetas diferentes. Solo puede utilizar un fichero ".htpasswd" para varios ".htaccess".

```

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.es-us.md
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_protect_directory_by_password/guide.es-us.md:1]: Please duplicate suggestions from the es-es file onto this es-us file.

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-ca.md [hunk 7]
```diff
@@ -64,8 +64,8 @@ Nous allons vous expliquer les 4 étapes principales à réaliser pour protéger
 > 
 > Pour modifier la version de PHP de vos sites web sur votre hébergement web, consultez les guides suivants :
 > 
-> - [Modifier la configuration de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
-> - [Changer la version de PHP de son hébergement web](/pages/web/hosting/php_configure_php_on_your_web_hosting_2014/)
+> - [Modifier la configuration de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime)
+> - [Changer la version de PHP de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-ca.md]: are you sure about this link ? (same as the one right above)

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-fr.md [hunk 8]
```diff
@@ -64,8 +64,8 @@ Nous allons vous expliquer les 4 étapes principales à réaliser pour protéger
 > 
 > Pour modifier la version de PHP de vos sites web sur votre hébergement web, consultez les guides suivants :
 > 
-> - [Modifier la configuration de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
-> - [Changer la version de PHP de son hébergement web](/pages/web/hosting/php_configure_php_on_your_web_hosting_2014/)
+> - [Modifier la configuration de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime)
+> - [Changer la version de PHP de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-fr.md]: See comment on fr-ca above

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.pt-pt.md
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_protect_directory_by_password/guide.pt-pt.md:1]: Title and excerpt are not updated in PT and updated in other languages. Please check.

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-ca.md (hunk 7 above)
Comment (Kazarus4) [reply pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-ca.md]: /pages/web/hosting/php_configure_php_on_your_web_hosting_2014

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-fr.md (hunk 8 above)
Comment (Kazarus4) [reply pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-fr.md]: /pages/web/hosting/php_configure_php_on_your_web_hosting_2014

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.es-us.md
Comment (Kazarus4) [reply pages/web/hosting/htaccess_protect_directory_by_password/guide.es-us.md:1]: changed

File: pages/web/hosting/htaccess_protect_directory_by_password/guide.pt-pt.md
Comment (Kazarus4) [reply pages/web/hosting/htaccess_protect_directory_by_password/guide.pt-pt.md:1]: done

**Comment by Kazarus4** [comment]
It stay to check ENGB and DE, duplication for EN subs and after all merge.

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**